tests/zipkin_test.py

```diff
@@ -377,6 +389,74 @@ def test_span_context(
     assert client_span == expected_client_span
 
 
+@mock.patch('py_zipkin.thread_local._thread_local', autospec=True)
+@mock.patch('py_zipkin.zipkin.generate_random_64bit_string', autospec=True)
+@mock.patch('py_zipkin.zipkin.zipkin_logger', autospec=True)
+def test_client_span_context(
+    zipkin_logger_mock,
+    generate_string_mock,
+    thread_local_mock,
+):
+    zipkin_attrs = ZipkinAttrs(
```

Comment:
Could you pull the first many lines of this test case into a helper function or a fixture? It's so very DRY.